### PR TITLE
{bio}[GCC/11.2.0] RAxML-NG v1.1.0

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/r/RAxML-NG/RAxML-NG-1.1.0-GCC-11.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+}]
+checksums = [None]
+
+builddependencies = [
+    ('CMake', '3.21.1'),
+    ('Bison', '3.7.6'),
+    ('flex', '2.6.4'),
+]
+
+separate_build_dir = True
```
Comment (jfgrimm) [inline easybuild/easyconfigs/r/RAxML-NG/RAxML-NG-1.1.0-GCC-11.2.0.eb:36]: not needed, this is the default behaviour
```suggestion
```

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
himem03.pri.viking.alces.network - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6130 CPU @ 2.10GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/6b9fef5f215e1625ecddabcfc229271a for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
himem03.pri.viking.alces.network - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6130 CPU @ 2.10GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/9405766b949a117060fae6c654e779e1 for a full test report.

**Review by jfgrimm — APPROVED** [review]
LGTM

**Comment by jfgrimm** [comment]
Going in, thanks @boegel!

**Pull request merged**